examples/run_over_exo.sh

```
@@ -0,0 +1,68 @@
+    done
+    echo ""
+
+    if ! curl -sf http://localhost:52415/v1/models >/dev/null 2>&1; then
+        echo "Exo failed to start"
+        kill $exo_pid 2>/dev/null
+        source "$HOME/.venv/bin/activate"
+        return 1
+    fi  
+
+    # Load model instance via placement API 
+    echo "Loading model instance: $exo_model"
```

Shellcheck (0.11.0)

[warning] exo_model is referenced but not assigned.

(SC2154)